tools/build_browser.js

```diff
@@ -14,7 +14,8 @@ const log = (...args) => console.log(...args);
 const { rollupCode } = require("./lib/bundling.js");
 const bundling = require('./lib/bundling.js');
 const Table = require('cli-table');
-const { result } = require('lodash');
+
+const textEncoder = new TextEncoder();
```

Comment:
Why is this necessary? Rollup is supposed to be generating ASCII.

Reply:
I'm sure. See `ascii_only: true` in build config.

It's also possible that `length` would work now where-as it didn't work before... the whole reason was so that the byte counts match the size on disk (which at the time they did not)... but I don't think we need this added TextEncoder stuff...

Side note: You don't need to rebase constantly either, it'd be easier for me to see what changes as we work on something if you just lay new commits on top of old ones.

Reply:
I'm +1 to remove the `TextEncoder` stuff, it's way easier to deal with strings. I'd like to note that `ascii_only: true` is a terser-only option, so the non-minified files may contain arbitrary UTF-8 chars. Not really an issue since the size that really matters is the gziped one and this one is reported appropriately.